administrator/components/com_fields/helpers/internal.php

```diff
@@ -2,17 +2,30 @@
 /**
  * @package     Joomla.Administrator
  * @subpackage  com_fields
- * 
+ *
  * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  */
 defined('_JEXEC') or die;
 
 JLoader::register('FieldsHelper', JPATH_ADMINISTRATOR . '/components/com_fields/helpers/fields.php');
 
+/**
+ * Fields component helper.
+ *
+ * @since  3.6
+ */
 class FieldsHelperInternal
 {
-
+	/**
+	 * Configure the Linkbar.
+	 *
+	 * @param   string  $context  The context of the content passed to the helper
+	 *
+	 * @return  void
+	 *
+	 * @since   1.6
```

Comment:
Yes. But the since tag told you when this method gets introduced. And this specific method will be introduced with 3.6 :)